packages/cross_file/pubspec.yaml

```diff
@@ -1,17 +1,17 @@
 name: cross_file
 description: An abstraction to allow working with files across multiple platforms.
 homepage: https://github.com/flutter/plugins/tree/master/packages/cross_file
-version: 0.3.0-nullsafety
+version: 0.3.0
 
 dependencies:
   flutter:
     sdk: flutter
-  meta: ^1.3.0-nullsafety.3
+  meta: ^1.3.0
 
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  pedantic: ^1.10.0-nullsafety.3
+  pedantic: ^1.10.0
 
 environment:
   sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Please update this the lower bound to `2.12.0-259.9.beta`

Reply:
I indeed remember this being discussed on Discord but wasn't able to find this in other packages I used as reference (i.e. meta and pedantic). Thanks for the update, I will stick to this convention.